compose.go

```diff
@@ -76,6 +80,135 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	composeCmd.Flags().BoolVar(&conflictsVerbosity, "conflicts-verbosity", false, "Log files conflicts")
 	composeCmd.Flags().BoolVar(&clean, "clean", false, "Remove .compose dir on start")
 	composeCmd.Flags().BoolVar(&interactive, "interactive", true, "Interactive mode allows to submit user credentials during action")
+
+	composeDependency := &compose.Dependency{}
+	strategies := &compose.RawStrategies{}
+	var createNew bool
+	var addCmd = &cobra.Command{
+		Use:   "compose:add",
+		Short: "Add a new package to plasma-compose",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return packagePreRunValidate(cmd, args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.AddPackage(createNew, composeDependency, strategies, p.wd)
+		},
+	}
+
+	var updateCmd = &cobra.Command{
+		Use:   "compose:update",
```

Comment:
Currently, user is prompted with cli form when passing some options to update command:
```
➜  ski-platform-bis git:(master) ./plasmactl compose:update --name plasma-coreee --tag 1.0.0
┃ Packages                                                                                                                                                                                     
┃ > pla-plasma                                                                                                                                                                                 
┃   plasma-core                                                                                                                                                                                
┃   plasma-coreee                                                                                                                                                                              
┃   plasma-knowledge                                                                                                                                                                           
┃   plasma-workspace                                                                                                                                                                           
```
Would be great for to have no user prompt when passing some options to update command and only update the fields that were specified, while leaving the rest of them un-touched

Reply:
Working as expected:
```
./plasmactl compose:update --package plasma-core --tag 1.0.0 --type git --url https://xxxxx.xxxxx.xxxxx/skilld/pla-plasma.git
./plasmactl compose:update --package plasma-core --tag 1.0.1000000
```